src/utils/api.js

```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
```

Comment:
Have a look at the notes on axios parma here: https://notes.northcoders.com/courses/js-front-end/react-projects as this will allow you to tidy up this function massively!